src/App.tsx

```diff
@@ -13,37 +15,55 @@ import { MessageContext } from "./contexts/MessageContextProvider";
 import { useTokenList } from "./hooks/tokenList";
 import { parseCSV, Payment } from "./parser";
 import { buildTransfers } from "./transfers";
+import { checkAllBalances, transfersToSummary } from "./utils";
 
 const App: React.FC = () => {
-  const { sdk } = useSafeAppsSDK();
+  const { safe, sdk } = useSafeAppsSDK();
+
   const { tokenList, isLoading } = useTokenList();
   const [submitting, setSubmitting] = useState(false);
   const [transferContent, setTransferContent] = useState<Payment[]>([]);
   const [csvText, setCsvText] = useState<string>(
     "token_address,receiver,amount,decimals"
   );
-  const { addMessage, setCodeWarnings } = useContext(MessageContext);
+  const { addMessage, setCodeWarnings, setMessages } =
+    useContext(MessageContext);
```

Comment:
My IDE format on save (using `esbenp.prettier-vscode`) disagrees with this style and changes it to 

```
  const { addMessage, setCodeWarnings, setMessages } = useContext(
    MessageContext
  );
  ```
  
  Everytime I touch this file. I have spent hours trying to get these to agree and it is really really annoying.
  
  Not only this, but the lint error I get is incredibly vague:
  
  ```
  yarn run v1.22.10
$ eslint --max-warnings 0 .

/Users/bensniff/Projects/bh2smith/safe-airdrop/src/App.tsx
  29:55  error  Replace `·useContext(⏎····MessageContext⏎··` with `⏎····useContext(MessageContext`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
  ```
  
  Any tips you have about this would allow me to continue working. Until then, I am afraid that I am fully blocked.

Reply:
Technically, I prefer what my IDE changes it to, but I would also be happy if they would just agree. I have been able to work around this by commenting the "react-app" extension in our eslintrc file

```
  extends: [
    "plugin:import/recommended",
    "plugin:import/typescript",
    "plugin:prettier/recommended",
    // "react-app",
  ],
  ```
  
  Still nowhere near figuring out what this exact style is called or how to refer to it in our configuration.

Reply:
I had some issues with this too. I think we should migrate everything to the lint settings of safe react apps so that we are consistent like you suggested in #59